code/modules/food_and_drinks/food/condiment.dm

```diff
@@ -227,6 +228,13 @@
 	list_reagents = list("mayonnaise" = 50)
 	possible_states = list()
 
+/obj/item/reagent_containers/condiment/yoghurt
+	name = "yoghurt"
```

Comment:
```suggestion
	name = "yoghurt tub"
```

Somewhere above you declared the container to be called a yoghurt tub, it's not reflected here